.jscsrc

```diff
@@ -2,5 +2,6 @@
   "preset": "airbnb",
   "disallowMultipleVarDecl": true,
   "requireMultipleVarDecl": null,
-  "requireDollarBeforejQueryAssignment": "ignoreProperties"
+  "requireDollarBeforejQueryAssignment": "ignoreProperties",
+  "requirePaddingNewLinesAfterBlocks":false
```

Comment:
There should be a space after the colon.

Reply:
It is fixed